### PR TITLE
Creates contributing documentation

### PR DESCRIPTION
**Pull request by cedancp-cs**
I have added some steps on how to create a branch and name conventions. 
Also added in the step two a suggestion on following conventional commits standard open for discussion.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+# Branch naming convention
+
+The branch name should include the category, the Jira ticket number and a short description using "-" as separator
+
+Use the following format: {category}/{Jira tickket number}-{short description}
+
+| Category  | Meaning                                             |
+|-----------|-----------------------------------------------------|
+| hotfix    | for quickly fixing critical issues                  |
```
Comment (LiuPierre) [inline CONTRIBUTING.md:20]: I am actually using 'commitizen' plugin on VSCode which is kind of similar to this but the naming is different. For example hotfix is 'fix'. There are also other namings such as 'refactor', 'doc', 'style'... 
I am up to follow the standard, do you guys use a plugin for it or just write prefix on every commit ?

Comment (cedancp-cs) [reply CONTRIBUTING.md]: made some small changes

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+Use the following format: {category}/{Jira tickket number}-{short description}.
+
+Follow conventional commit categories for branch naming:
+
+| Category  | Meaning                                             |
+|-----------|-----------------------------------------------------|
+| fix       | for fixing a bug                                    |
+| feat      | for adding, removing or modifying a feature         |
+| test      | for experimenting something which is not an issue   |
+| wip       | for a work in progress                              |
```
Comment (AldinaSculac) [inline CONTRIBUTING.md:25]: I'm not sure how often we will need to use the `wip` category, usually, we can add the "work in progress" label on a PR.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+1. Create a new branch from **main** branch following the name convention.
+2. Use [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) convention for commit messages.
+3. Create a PR and include any useful context of the change in the description. Include any relevant screenshots.
+4. Wait for the PR to be reviewed and approved.
+
+# Branch naming convention
+
+The branch name should include the category, the Jira ticket number and a short description using "-" as separator.
+
+Use the following format: {category}/{Jira tickket number}-{short description}.
```
Comment (AldinaSculac) [inline CONTRIBUTING.md:16]: You got one extra "k"  :)

File: CONTRIBUTING.md (hunk 2 above)
Comment (cedancp-cs) [reply CONTRIBUTING.md:25]: make sense, I'll remove it

File: CONTRIBUTING.md (hunk 3 above)
Comment (cedancp-cs) [reply CONTRIBUTING.md:16]: good catch!

Comment (AldinaSculac) [reply CONTRIBUTING.md:16]: Is this meant for people outside the CSQ to contribute?  If so, they will not have access to the JIRA board.

**Pull request closed without merging**